rules/cloud/azure/azure_app_credential_added.yml

```diff
@@ -0,0 +1,20 @@
+title: Added credentials to existing application
```

Comment:
`title: Added Credentials to Existing Application` https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#common-pitfalls